src/go/rpk/pkg/cli/cmd/topic/consume.go

```diff
@@ -488,7 +492,11 @@ func parseConsumeTimestamp(
 		if negate {
 			rel = -rel
 		}
-		at = time.Now().Add(rel)
+		if len(t0) > 0 {
+			at = t0[0].Add(rel).UTC()
+		} else {
+			at = time.Now().Add(rel).UTC()
+		}
```

Comment:
I think this can be `t0.Add(rel).UTC()` always, and the caller can put in time.Now or t0

Reply:
Smart, thank you!